assignment02/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title> Assignment 2 </title>
+    <meta charset="utf-8">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="js/myScript2.js"></script>
+</head>
+
+<body>
+    <!--
+        James Welu
+        assignment02.html
+        INFO2340.WW
+        Thoendel
+        12/11/2019
+    -->
+
+
+<h1>The Wizard of OZ!</h1>
+
+
+<p><strong><span id="female1"></span></strong> lived in the midst of the great <strong><span in="state"></span></strong></p>

```

Comment:
You've got some extra paragraph tags in here - but you've done a nice job with the span tags and I names! Looks like you used some numbering to keep things straight. Nice!

Reply:
I guess my logic was that it would read more like a book. It was also easier to keep track of when broken down into small segments.